src/test/java/uk/gov/hmcts/cmc/claimstore/courtfinder/models/AddressTest.java

```diff
@@ -1,6 +1,6 @@
 package uk.gov.hmcts.cmc.claimstore.courtfinder.models;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
```

Comment:
This could be reverted just to keep this intro PR clean, otherwise let's format it consistently (remove `public`)

Reply:
you are right, i will remove public...

when i started this work, I thought i will take easy route and take smallest test :) but decided to add more later. 

I will leave this too..